app/views/users/circuitverse/edit.html.erb

```diff
@@ -77,8 +77,13 @@
       .attr('src', e.target.result);
     };
     reader.readAsDataURL(input.files[0]);
+    document.getElementById("remove_pic").value = "0";
     }
   }
+  function rempic(){
```

Comment:
This won't remove the pic if the user click on the button and refresh the page, also the image path given below won't work in production

Reply:
I have suggestion. What if we can implement a popup for profile image similar as we have in social media.

We can remove the scope of editing profile image from edit.

WDYT?